### PR TITLE
fix dependency

### PR DESCRIPTION
**Pull request by dustjr1565**
numpy version 
fixed 1.23.5

reference: https://github.com/jdb78/pytorch-forecasting/issues/1236

A dependency issue has been identified between the numpy and torchmetric libraries.
When the numpy version is 1.24 or higher, a dependency issue arises with torchmetric version 1.0.0 or higher.
Switching the numpy version to 1.23.5 resolves the issue and allows for normal operation

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -23,6 +23,8 @@ timm==0.6.13
 autocare-dlt==0.2.4
 
 # etc
+numpy<=1.23.5
+pillow==9.5
```
Comment (Jiyong-Jeon) [inline requirements.txt:27]: We have already downgraded the pillow version from #255.
Please check if there is a conflict.

**Review by Jiyong-Jeon — APPROVED** [review]

File: .github/workflows/ddp.yaml [hunk 2]
```diff
@@ -13,9 +13,9 @@ jobs:
       matrix:
         python-version: ['3.9']
         pytorch-version: [
-          'torch==1.13.1+cu117 torchvision==0.14.1+cu117 --extra-index-url https://download.pytorch.org/whl/cu117',
-          'torch==2.0.0+cu117 torchvision==0.15.1+cu117 --index-url https://download.pytorch.org/whl/cu117',
-          'torch==2.0.1+cu117 torchvision==0.15.2+cu117 --index-url https://download.pytorch.org/whl/cu117'
+          'torch==1.13.1+cu117 torchvision==0.14.1+cu117 torchaudio==0.13.1 --extra-index-url https://download.pytorch.org/whl/cu117',
+          'torch==2.0.0+cu117 torchvision==0.15.1+cu117 torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cu117',
+          'torch==2.0.1+cu117 torchvision==0.15.2+cu117 torchaudio==2.0.2+cu117 --index-url https://download.pytorch.org/whl/cu117'
```
Comment (ZeroAct) [inline .github/workflows/ddp.yaml:18]: Do we need torchaudio?

Comment (dustjr1565) [reply .github/workflows/ddp.yaml:18]: it seems that you are importing torchaudio in torchmetric.
 Previously, there was a failure in the action due to version mismatch between torchaudio and torch.

Comment (dustjr1565) [reply .github/workflows/ddp.yaml:18]: Additionally, the same issue was observed on Windows, and it was resolved by changing the versions of torchaudio, numpy, and numba.

**Pull request merged**